src/test/groovy/org/prebid/server/functional/model/request/auction/Prebid.groovy

```diff
@@ -16,6 +16,7 @@ class Prebid {
     Amp amp
     Channel channel
     Map<String, BidderName> aliases
+    Map<String, Integer> aliasgvlids
```

Comment:
Please use camelCase for fields. To properly map different styles, use Jackson annotations.

Reply:
I think it's not necessary, because in ExtRequestPrebid the same naming https://github.com/prebid/prebid-server-java/blob/8ecf10089c1c2b6a5f9ef0c6f6ad60042fd5716f/src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRequestPrebid.java#L40